ACE/include/makeinclude/platform_macosx.GNU

```diff
@@ -42,10 +44,15 @@ else ifeq ($(MACOS_MAJOR_VERSION),11)
     ## just use the latest known version
     MACOS_CODENAME = $(MACOS_CODENAME_VER_11_latest)
   endif
+else ifeq ($(MACOS_MAJOR_VERSION),12)
+  ifeq ($(shell test $(MACOS_MINOR_VERSION) -gt 2; echo $$?),0)
```

Comment:
Shouldn't this be `-gt 0`

Reply:
Indeed, you are correct. I have changed the comparison.